chromium_presubmit_config.json5

```diff
@@ -163,10 +163,6 @@
       "android/java/org/chromium/chrome/browser/qrreader/CameraSource\\.java",
       "browser/notifications/android/java/src/org/chromium/chrome/browser/notifications/BraveNotificationManagerProxyImpl\\.java",
     ],
-    "/#include <random>": [
-      "components/brave_rewards/core/common/security_util\\.cc",
-      "components/brave_rewards/core/legacy/bat_helper\\.cc",
```

Comment:
do we actually need `<random>` in these files? I don't see any usage.

Reply:
Doesn't look like we do. Removed includes in 2113cfa08b9